Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```diff
@@ -15,18 +14,35 @@
     )
 
     Try {
+
+        $Domain = "."
+        $TestUser = ""
+        $DomainUser = $TestUserName.Split("\")
+        if ($DomainUser.Length -gt 1) {
+            $Domain = $DomainUser[0]
+            $TestUser = $DomainUser[1]
+        }
+        else {
+            $TestUser = $TestUserName
+        }
+        if ($Domain -eq ".") {
+            $Domain = $Env:COMPUTERNAME
+        }
+
+        Write-Verbose -Message "Test User $TestUser" -Verbose
+        Write-Verbose -Message "Test UserDomain $Domain" -Verbose
```

Comment:
Can it info message?

Reply:
Ideally this logic can moved right b4 using it.. Only in interactive process code

Reply:
No info messages are allowed in Remote PS. However all these messages will be visible as info in build logs